File: 2023/UICPC Regional/Round #9/solutions/fizzbuzz/submissions/accepted/fizzbuzz_zf.cpp
#include <iostream>

using namespace std;

int main() {
    int x, y, n;
    cin >> x >> y >> n;

    for (int i = 1; i <= n; ++i) {
        int div = 0;
        if (i%x == 0) {
            ++div;
            cout << "Fizz";
        }
        if (i%y == 0) {
            ++div;
            cout << "Buzz";
        }
        if (div == 0) cout << i;
        cout << endl;
    }
    return 0;
}

File: 2023/UICPC Regional/Round #9/solutions/fizzbuzz/submissions/accepted/fizzbuzz_zf.py
#!/usr/bin/python3

x,y,n = map(int, input().split())

for i in range(1,n+1):
    out = ""
    if i%x == 0: out = "Fizz"
    if i%y == 0: out += "Buzz"
    if out == "": out = i
    print(out)
